archeryutils/classifications/agb_outdoor_classifications.py

```diff
@@ -54,6 +65,9 @@ def _make_agb_outdoor_classification_dict() -> Dict[str, Dict[str, Any]]:
     ArcheryGB 2023 Rules of Shooting
     ArcheryGB Shooting Administrative Procedures - SAP7 (2023)
     """
+    # Five too many locals, but better than repeated dictionary assignment => disable
+    # pylint: disable=too-many-locals
+
```

Comment:
Yes, I think this is much more readable.
Previous code made sense at the time as we were developing the system, but a fresh/alternative pair of eyes is good.
The difficulty of 'scientific' software vs. clearly-defined output software...